packages/page-accounts/src/Accounts/modals/Create.tsx

```diff
@@ -306,31 +301,9 @@ function Create ({ className = '', onClose, onStatusChange, seed: propsSeed, typ
             <p>{t<string>('The secret seed value for this account. Ensure that you keep this in a safe place, with access to the seed you can re-create the account.')}</p>
           </Modal.Column>
         </Modal.Columns>
-        <Modal.Columns>
-          <Modal.Column>
-            <Password
-              className='full'
-              help={t<string>('This password is used to encrypt your private key. It must be strong and unique! You will need it to sign transactions with this account. You can recover this account using this password together with the backup file (generated in the next step).')}
-              isError={!isPassValid}
-              label={t<string>('password')}
-              onChange={_onChangePass}
-              onEnter={_onCommit}
-              value={password}
-            />
-            <Password
-              className='full'
-              help={t<string>('Verify the password entered above.')}
-              isError={!isPass2Valid}
-              label={t<string>('password (repeat)')}
-              onChange={_onChangePass2}
-              onEnter={_onCommit}
-              value={password2}
-            />
-          </Modal.Column>
-          <Modal.Column>
-            <p>{t<string>('The password and password confirmation for this account. This is required to authenticate any transactions made and to encrypt the keypair.')}</p>
-          </Modal.Column>
-        </Modal.Columns>
+        <PasswordInput onChange={_onPasswordChange}
+          onEnter={_onCommit}
+          password={password}/>
```

Comment:
```suggestion
        <PasswordInput
          onChange={_onPasswordChange}
          onEnter={_onCommit}
          password={password}/>
```

Not sure why the linter didn't complain here.